lib/src/alert/Alert.tsx

```diff
@@ -122,7 +121,7 @@ const calculateWidth = (margin, size) =>
     ? `calc(${sizes[size]} - ${getMargin(margin, "left")} - ${getMargin(margin, "right")})`
     : sizes[size];
 
-const AlertModal = styled.div`
+const AlertModal = styled.div<{ mode: "inline" | "modal" }>`
```

Comment:
As we discuss, extracting the types from the `ComponentPropsType` may be a better approach for us:

![image](https://user-images.githubusercontent.com/44321109/231700634-c546aa65-5c5c-4a30-a811-8681308b2cde.png)

Reply:
This would reduce the difficulty and cost of updating the type of a prop.

Reply:
Totally agree! I will change it in all our components.